azurerm/internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -510,10 +528,10 @@ func expandSqlVirtualMachineDataStorageSettings(input []interface{}) *sqlvirtual
 }
 
 func expandSqlVirtualMachineStorageSettingsLuns(input []interface{}) *[]int32 {
-	expandedLuns := make([]int32, len(input))
+	expandedLuns := make([]int32, 0, len(input))
```

Comment:
Not sure we need the capacity arg here since we're ranging over `input`?

Reply:
OK cool, yeah it works without it too.